### PR TITLE
update kcc to use -profile and -v for switching

### PR DESCRIPTION
**Pull request by dwightguth**
@chathhorn @grosu please review

### CODE REVIEW

File: scripts/kcc [hunk 1]
```diff
@@ -26,14 +26,24 @@ my @srcFiles = ();
 my @tempFiles = ();
 
 my $distDirectory = dirname(rel2abs($0));
+our $profile;
```
Comment (chathhorn) [inline scripts/kcc]: I notice some others have crept in as well, but shouldn't all the variables in this file be declared using "my" so they're lexically scoped to this file?

**Comment by chathhorn** [comment]
Looks fine to me otherwise, assuming the tests pass.

File: scripts/kcc (hunk 1 above)
Comment (dwightguth) [inline scripts/kcc]: @chathhorn this is done so that they can be accessed from the actions on the Getopt::Declare.

**Comment by dwightguth** [comment]
The build is failing because of a bug with the makefile that should be fixed by the installer pull request. All tests are actually passing, so I am going to merge this.

**Pull request merged**